docs/messaging.rst

```diff
@@ -859,7 +859,9 @@ multiple cases:
 
 The client sends a shutdown request to the kernel, and once it receives the
 reply message (which is otherwise empty), it can assume that the kernel has
-completed shutdown safely.
+completed shutdown safely.  The request can be sent on either the `control` or
+`shell` channels.  The reply is made both to the requesting channel and to
+the `iopub` channel.
```

Comment:
Is it? @minrk is it intentional that a `*_reply` message is broadcast over iopub? I can understand that it makes sense for all clients to get a notification that shutdown is underway, but I'm a bit surprised we do it by broadcasting the `shutdown_reply`.